client/src/components/Search/Search.js

```diff
@@ -183,12 +182,12 @@ class Search extends Component {
   /**
    * Try to find the first form field in the advanced form and focus on it.
    */
-  focusFirstFormField(filter = 'input, textarea, select, button') {
+  focusFirstFormField(filter = 'input:not([type=hidden]), textarea, select, button') {
```

Comment:
Why was this change made? It doesn't seem related to the main issue?

Reply:
First field in "Search options" block is hidden. Because of this, when we open "Search options" block the first field is not in focus.

Reply:
Is that somehow related to the issue this PR is for? Or is it separate? If it's separate please handle it in a separate issue and PR to make it clearer what's going on.

Reply:
It doesn't work and you cannot test focus on first field in "Search options". I can extract it to another issue if so would be better.

Reply:
Updated.
PR https://github.com/silverstripe/silverstripe-admin/pull/1567